app/views/users/_users_list.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="columns is-multiline">
+  <% users.each_with_index do |user, index| %>
+    <div class="tile is-parent column is-one-quarter">
+      <%= render partial: "user_card", locals: {user: user} %>
+    </div>
+  <% end %>
+</div>
```

Comment:
@poofyOwl I've modified this slightly to use Bulma's multiline grid so we don't need to calculate the number of cards per row :)